OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/PgpSignEncryptOperation.java

```diff
@@ -471,7 +527,7 @@ public PgpSignEncryptResult execute(PgpSignEncryptInputParcel input, CryptoInput
 
                 if (enableCompression) {
                     // Use preferred compression algo
-                    int algo = input.getCompressionAlgorithm();
+                    int algo = data.getCompressionAlgorithm();
```

Comment:
for the future, please avoid refactorings like this, or put them in a separate commit. they add a lot of lines to the review which contain no semantic changes